### ISSUE TITLE
Modal view: color of icons in the navigation bar are blue instead of red

### ISSUE THREAD

**Issue by benoitchantre** [Simbul/baker#1137]
To have a better coherence through the app, there should be a unique color for interactive elements of the UI.

**Comment by benoitchantre** [comment]
@folletto I can submit a PR to fix that.
What is the better place to define the color of icons in the modal view: in `UIConstants.h`?
If yes, should we move `UIConstants.h` in the Baker directory (more neutral than BakerShelf directory)?

**Comment by folletto** [comment]
I haven't looked into it yet so I don't have a direct answer, but as a general direction I would for now just use the same constant we already use for the other parts of the UI and keep `UIConstants.h` in place (this second part mostly because will break the project file and that's a very annoying thing to deal with).

Should be `ISSUES_ACTION_BUTTON_BACKGROUND_COLOR` if I recall correctly.

**Comment by benoitchantre** [comment]
`ISSUES_ACTION_BUTTON_BACKGROUND_COLOR` is used for the download/read button for issues in the shelf.

``` Objective-C
    // Background color for issues cover (before downloading the actual cover)
    #define ISSUES_COVER_BACKGROUND_COLOR @"#ffffff"

    // Title for issues in the shelf
    // #define ISSUES_TITLE_FONT @"Helvetica"
    // #define ISSUES_TITLE_FONT_SIZE 15
    #define ISSUES_TITLE_COLOR @"#000000"

    // Info text for issues in the shelf
    // #define ISSUES_INFO_FONT @"Helvetica"
    // #define ISSUES_INFO_FONT_SIZE 15
    #define ISSUES_INFO_COLOR @"#929292"

    #define ISSUES_PRICE_COLOR @"#b72529"

    // Download/read button for issues in the shelf
    // #define ISSUES_ACTION_BUTTON_FONT @"Helvetica-Bold"
    // #define ISSUES_ACTION_BUTTON_FONT_SIZE 11
    #define ISSUES_ACTION_BUTTON_BACKGROUND_COLOR @"#b72529"
    #define ISSUES_ACTION_BUTTON_COLOR @"#ffffff"

    // Archive button for issues in the shelf
    // #define ISSUES_ARCHIVE_BUTTON_FONT @"Helvetica-Bold"
    // #define ISSUES_ARCHIVE_BUTTON_FONT_SIZE 11
    #define ISSUES_ARCHIVE_BUTTON_COLOR @"#b72529"
    #define ISSUES_ARCHIVE_BUTTON_BACKGROUND_COLOR @"#ffffff"

    // Text and spinner for issues that are being loaded in the shelf
    #define ISSUES_LOADING_LABEL_COLOR @"#b72529"
    #define ISSUES_LOADING_SPINNER_COLOR @"#929292"

    // Progress bar for issues that are being downloaded in the shelf
    #define ISSUES_PROGRESSBAR_TINT_COLOR @"#b72529"
```

Do you want I use this constant or a/some new one(s)?

**Comment by folletto** [comment]
I can't check now, but it's simple: it should be the same one that gives color to the Navigation Bar. :)

**Comment by benoitchantre** [comment]
You were right:

``` objective-c
[navigationBar setTintColor:[UIColor colorWithHexString:ISSUES_ACTION_BUTTON_BACKGROUND_COLOR]];
```

Thanks

**Comment by benoitchantre** [comment]
Fix: #1149
